schemas/common/schemadescriptor.schema.json

```diff
@@ -0,0 +1,75 @@
+{
+  "meta:license": [
+    "Copyright 2017 Adobe Systems Incorporated. All rights reserved.",
+    "This work is licensed under a Creative Commons Attribution 4.0 International (CC BY 4.0) license",
+    "you may not use this file except in compliance with the License. You may obtain a copy",
+    "of the License at https://creativecommons.org/licenses/by/4.0/"
+  ],
+  "$id": "https://ns.adobe.com/xdm/common/schemadescriptor",
+  "$schema": "http://json-schema.org/draft-06/schema#",
+  "title": "Schema Descriptor",
+  "meta:extensible": true,
```

Comment:
Do we need extensibility here?

Reply:
Hi @trieloff, yes I believe that we do want to support extensibility of schemas descriptors.  The model here would be to create a new descriptor type (via @type), and extend this schema to include any additional properties needed by the new descriptor type.